src/app/globals.scss

```
@@ -266,8 +266,8 @@
     width: 56px;
     height: 56px;
     padding: 8px;
-    top: 50%;
-    transform: translateY(-200%);
+    top: 40vh; /* 40% of viewport height instead of 50% */
```

Comment nice for the review, but not useful later.
```suggestion
    top: 40vh;
```